aplt/scenarios.py

```diff
@@ -0,0 +1,26 @@
+"""Several basic push scenarios"""
+from twisted.python import log
+
+from aplt.commands import (
+    connect,
+    hello,
+    register,
+    send_notification,
+    expect_notification,
+    unregister,
+    disconnect,
+    ack,
+    random_channel_id,
+)
+
+
+def basic():
```

Comment:
Yep, have a rather large amount of docs coming in a later PR on how to do everything.